sdv/tabular/base.py

```diff
@@ -334,6 +334,37 @@ def _make_conditions_df(self, conditions, num_rows):
 
         return conditions.copy()
 
+    def _conditionally_sample_rows(self, dataframe, max_retries, max_rows_multiplier,
+                                   condition, transformed_condition, float_rtol,
+                                   graceful_reject_sampling):
+        sampled_rows = self._sample_batch(
+            len(dataframe),
+            max_retries,
+            max_rows_multiplier,
+            condition,
+            transformed_condition,
+            float_rtol
+        )
+
+        if len(sampled_rows) < len(dataframe):
```

Comment:
VERY minor thing: we could capture the `len(sampled_rows)` and `len(dataframe)` in variables, so we do not need to compute them multiple times. It will also help a bit on the readability.